website/src/tabs/transaction/SendCredits.js

```diff
@@ -0,0 +1,91 @@
+import React, {useState} from "react";
+import {Card, Divider, Form, Input, Button } from "antd";
+const { TextArea } = Input;
+import {useAleoWASM} from "../../aleo-wasm-hook";
+import {downloadAndStoreFiles, getAllSavedFiles} from '../../db';
+
+export const SendCredits = () => {
+    const [privateKey, setPrivateKey] = useState("APrivateKey1zkp3dQx4WASWYQVWKkq14v3RoQDfY2kbLssUj7iifi1VUQ6");
+    const [toAddress, setToAddress] = useState("aleo184tj0fllfuzqzpmw5jt6l2ptx0avhjxh95u9llcr6ypf6fx3hvrsref0ju");
+    const [amount, setAmount] = useState(50);
+    const [plaintext, setPlaintext] = useState(`{
+      owner: aleo184vuwr5u7u0ha5f5k44067dd2uaqewxx6pe5ltha5pv99wvhfqxqv339h4.private,
+      gates: 1159017656332810u64.private,
+      _nonce: 1635890755607797813652478911794003479783620859881520791852904112255813473142group.public
+    }`);
+    const [transaction, setTransaction] = useState(null);
+    const aleo = useAleoWASM();
+
+    const safeStateUpdate = (update, event) => {
+      try { update(event.target.value) }
+      catch (error) { console.error(error)}
+    }
+
+    const buildTransaction = async () => {
+      try {
+        let startTime = performance.now();
+        await downloadAndStoreFiles();
+        console.log(`Download Completed: ${performance.now() - startTime} ms`);
+        startTime = performance.now();
+
+        const files = await getAllSavedFiles();
```

Comment:
Maybe these can be appropriately filtered before js?

Reply:
Yeah, this was lazy. I'll update it.